skada/tests/test_subspace.py

```diff
@@ -48,10 +55,13 @@ def test_subspace_alignment(estimator, da_dataset):
     "adapter, n_samples, n_features, n_components",
     [
         (SubspaceAlignmentAdapter(), 5, 3, 3),
-        (SubspaceAlignmentAdapter(), 2, 5, 2),
+        (SubspaceAlignmentAdapter(), 3, 3, 3),
         (TransferComponentAnalysisAdapter(), 5, 3, 3),
         (TransferComponentAnalysisAdapter(), 2, 3, 3),
         (TransferComponentAnalysisAdapter(), 2, 5, 4),
+        (TransferJointMatchingAdapter(), 5, 3, 3),
+        (TransferJointMatchingAdapter(), 2, 3, 3),
+        (TransferJointMatchingAdapter(n_components=4), 2, 5, 4),
```

Comment:
This test is made to check that the right `n_components` is taken for `n_component=None`.